pkg/virt-handler/network/cniproxy/cniproxy.go

```diff
@@ -0,0 +1,150 @@
+package cniproxy
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/golang/glog"
+
+	"kubevirt.io/kubevirt/pkg/virt-handler/utils"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+//TODO: Make it configurable
+const (
+	CNINetDir     = "/etc/cni/net.d"
+	CNIPluginsDir = "/opt/cni/bin"
+	LibvirtSocket = "/var/run/libvirt/libvirt-sock"
+)
+
+type CNIProxy struct {
+	cniConfig   *libcni.CNIConfig
+	netConfig   *libcni.NetworkConfig
+	runtimeConf *libcni.RuntimeConf
+}
+
+func confFiles(dir string, extensions []string) ([]string, error) {
+	// In part, adapted from rkt/networking/podenv.go#listFiles
+	files, err := ioutil.ReadDir(dir)
+
+	switch {
+	case err == nil: // break
+		glog.Errorln("err==nil")
```

Comment:
Out of curiosity, why use glog here?

Using the KubeVirt logging wrapper, we'll get structured logging for "free".

Reply:
Thanks, yes, I've changed it all in the next patch set.